src/api/gnosis.js

```diff
@@ -66,7 +69,7 @@ export const initGnosisConnection = async (GNOSIS_OPTIONS) => {
 }
 
 export const initReadOnlyGnosisConnection = async (GNOSIS_OPTIONS) => {
-  if (gnosisROInstance) return
+  if (gnosisInstances.readOnly) return
```

Comment:
Can this function maybe be generic? 🙈 If the function signature worked like this:
`const initGnosisConnection = type => async (...```

Just like the `waitForGnosisConnection` function